src/electos/ballotmaker/constants.py

```diff
@@ -4,7 +4,7 @@
 import errno
 
 PROGRAM_NAME = "BallotMaker"
-VERSION = "0.1.1"
+VERSION = "0.1.2"  # incremented September 7, 2022
```

Comment:
I'm a [-0](https://www.apache.org/foundation/voting.html#expressing-votes-1-0-1-and-fractions) on this. The date status is a nice thing to see. It has the downside is that it's one more step to update when the version changes. It's slightly less bookkeeping to defer to the changelog as the source of the dates.

Reply:
In fact, I found the dates for the changelog by reviewing the Git history for this line. Plus, I've got a little keyboard shortcut on my Mac that types out the date. If it's a hassle I'll delete it.